### ISSUE TITLE
Windows Defender is blocking the installation

### ISSUE THREAD

**Issue by adrianoenache** [responsively-org/responsively-app#587]
# 🐞 bug report

### ✍️ Description

Windows Defender is blocking the installation.

![Capturar](https://user-images.githubusercontent.com/3247230/118338793-66361000-b4ed-11eb-965b-726693422ad7.PNG)

Someone know why? 

This is not good for the brand.

I installed it before and this not happened, but I don't remember the version.

I do not feel secure installing it.

**Comment by jjavierdguezas** [comment]
related to #151 

shouldn't this be fixed by now @manojVivek ?

@adrianoenache we already code signed the app, it shouldn't be showing that warning by now but it is not in our hands. The certificate that ensures an immediate bypass of the Windows SmartScreen is **very expensive**. We opted for a cheaper one that has an "organic reputation build" so it is going to disappear _eventually_ 

![image](https://user-images.githubusercontent.com/13673443/118339797-37825e80-b51a-11eb-9209-219bc792100f.png)

https://twitter.com/jotajota930621/status/1308469353701142531?s=19

**Comment by manojVivek** [comment]
> shouldn't this be fixed by now @manojVivek ?

Yeah JJ, ideally it should have. But we never know how the reputation system works to be sure. 😐

**Comment by brainz80** [comment]
Having the same issue - though in my case the "Run anyway" button doesn't exist. Probably a case of policy settings.

Was able to "install" anyway by using 7zip to unpack the installer and then unpacking `$PLUGINSDIR/app-64.7z` - this file contains the app.

**Comment by codigoisaac** [comment]
I just made the download from the website and got the same issue...
